src/context/ToastContext.jsx

```diff
@@ -0,0 +1,43 @@
+import { v4 as uuid } from "uuid";
```

Comment:
suggestion:  you can avoid big library for small use case by writing a simple id generator, id can be just timestamp